python/seldon_core/user_model.py

```diff
@@ -4,12 +4,19 @@
 from typing import Dict, List, Union, Iterable, Callable, Optional
 import numpy as np
 from seldon_core.proto import prediction_pb2
+from seldon_core.metrics import SeldonMetrics
 import inspect
 import logging
+import os
 
 logger = logging.getLogger(__name__)
 
 
+CLIENT_GETS_METRICS = (
+    os.environ.get("INCLUDE_METRICS_IN_CLIENT_RESPONSE", "true").lower() == "true"
```

Comment:
I guess `operator` could set this variable by to `true` or `false` depending if it is deploying `engine` or `executor`.

Reply:
Actually, this is independent from executor / engine usage and mostly depend if client expect to get those or not.

Reply:
Let's keep as it is for now, defaulting to current behaviour, and we can change it on a future release.

On the other hand, regarding the name of the variable, I think that it could be confusing to understand where does the variable `CLIENT_GETS_METRICS` come from out of context.

Reply:
Leaving as `true` for now due to backward compatibility reasons.

Just as a side note: Despite the choice custom metrics will anyway be exposed to Prometheus on wrapper's endpoint.